README.md

```diff
@@ -22,7 +22,7 @@ commits and tags:
 * works without any VCS, but happily reads tag information from and writes
   commits and tags to Git and Mercurial if available
 * just handles text files, so it's not specific to any programming language
-* supports Python 3 and PyPy3
+* supports non-EOL Python (>=3.6 at time of writing) and PyPy3
```

Comment:
Any particular reason to stop testing 3.5?  It's EOL at Python.org but some long-term-support distros might still support it.

Of course, enabling f-strings counts as a good reason ;-)